specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2018-11-19/machineLearningServices.json

```diff
@@ -1749,6 +1753,36 @@
         }
       }
     },
+    "AksNetworkingConfiguration": {

```

Comment:
do you need to update examples with this ?

Reply:
I currently don't have examples of any of the optional/advance properties right now. So, no updates in that area. I could add an example if needed.